docs/ref/contrib/contenttypes.txt

```diff
@@ -299,6 +299,11 @@ model:
        is ``True``. This mirrors the ``for_concrete_model`` argument to
        :meth:`~django.contrib.contenttypes.models.ContentTypeManager.get_for_model`.
 
+    .. attribute:: GenericForeignKey.allow_unsaved_instance_assignment
+
+       Works analogously to :attr:`ForeignKey.allow_unsaved_instance_assignment`.
```

Comment:
need `ForeignKey.allow_unsaved_instance_assignment <django.db.models.ForeignKey.allow_unsaved_instance_assignment>` here since we aren't in the django.db.models module of the docs.